<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 63, 95);
border-image: url(:/picetures/兔子耳朵.png);
border-image: url(:/picetures/背景 啊啊啊啊.png);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/picetures/白色.png);
border-radius:8px;
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);

color: rgb(48, 48, 48);
font: 75 14pt &quot;华康娃娃体W5&quot;;

border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <item>
     <property name="text">
      <string>按综合成绩升序排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按综合成绩降序排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按学分升序排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按学分降序排序</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);

color: rgb(48, 48, 48);
font: 75 16pt &quot;华康娃娃体W5&quot;;

border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>录入</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);
color: rgb(0, 0, 0);

font: 9pt &quot;ADMUI3Lg&quot;;

font: 75 14pt &quot;华康娃娃体W5&quot;;


</string>
    </property>
    <item>
     <property name="text">
      <string>课程列表</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学生列表</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成绩列表</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/picetures/白色.png);
border-radius:5px;
border: 1px solid blue;
color: rgb(6, 6, 6);
QLineEdit::hover {border: 1px solid blue;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);

color: rgb(48, 48, 48);
font: 75 14pt &quot;华康娃娃体W5&quot;;

border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>信息查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);

color: rgb(48, 48, 48);
font: 75 14pt &quot;华康娃娃体W5&quot;;

border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>成绩查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;华康娃娃体W5&quot;;
border-image: url(:/picetures/背景 蓝.png);




color: rgb(45, 45, 45);
</string>
    </property>
    <property name="text">
     <string>        学生成绩管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/picetures/关闭背景.png);
border-image: url(:/picetures/关闭.png);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);
color: rgb(48, 48, 48);
font: 75 16pt &quot;华康娃娃体W5&quot;;
border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);
color: rgb(48, 48, 48);
font: 75 16pt &quot;华康娃娃体W5&quot;;
border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/最终按钮.png);
color: rgb(48, 48, 48);
font: 75 16pt &quot;华康娃娃体W5&quot;;
border-radius:10px;
color: rgb(19, 30, 89);
</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

border-image: url(:/picetures/白色.png);

border-radius:5px;
border: 1px solid blue;

color: rgb(6, 6, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/白色.png);
border-image: url(:/picetures/学生信息.png);
font: 75 13pt &quot;华康娃娃体W5&quot;;
color: rgb(255, 255, 255);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>学号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(131, 131, 98);
color: rgb(163, 163, 122);
border-image: url(:/picetures/白色.png);

border-radius:5px;
color: rgb(6, 6, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picetures/白色.png);
border-image: url(:/picetures/课程信息.png);
font: 75 13pt &quot;华康娃娃体W5&quot;;
color: rgb(255, 255, 255);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>课程号：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
